app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :favorited_movies, through: :likes, source: :movie
+  has_many :watchlists
+  has_many :watchlist_movies, through: :watchlists, source: :movie
+
+  has_secure_password
+  validates :name, :email, presence: true
```

Comment:
Definitely need some sort of uniqueness validation for your users (either on email or name). Otherwise, some users may not be able to log in!